### PR TITLE
root resolver test

### PR DESCRIPTION
**Pull request by sacefe**

### CODE REVIEW

File: src/resolvers/rootResolver.test.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { getMockApolloServer } from "../__mocks__/mockApolloServer";
+import { createTestClient, ApolloServerTestClient } from "apollo-server-testing";
+import {gql} from "apollo-server-express";
+import logger from "../utils/logger";
+
+const createRootMutation = gql`
+    mutation {
```
Comment (teraSurfer) [inline src/resolvers/rootResolver.test.ts:7]: Make query and mutation uniform, either make mutation like this
```
mutation Root {
    root
}
```

or, make query like this
```
query {
    root
}
```

File: src/resolvers/rootResolver.test.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        });
+
+        it("Should create the root mutation", async() => {
+            const { data, errors } = await testApolloClient.mutate({
+                mutation: createRootMutation,
+            });
+            expect(errors).toBeFalsy();
+            expect(data.root).toBeTruthy();
+            expect(data.root).toEqual("Server is up.");
+            logger.info(data);
```
Comment (teraSurfer) [inline src/resolvers/rootResolver.test.ts]: We don't want logs in test cases, use it to review but don't push it to main.

File: src/resolvers/rootResolver.test.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            const testApolloClient = createTestClient(getMockApolloServer(mockContext));
+
+            const { data, errors } = await testApolloClient.query({
+                query: rootQuery
+            });
+
+            expect(errors).toBeFalsy;
+            expect(data.root).toBeTruthy;
+            expect(data.root).toEqual("Server is up.");
+            logger.info(data.root);
```
Comment (teraSurfer) [inline src/resolvers/rootResolver.test.ts]: Same as above comment.

**Review by teraSurfer — CHANGES_REQUESTED** [review]

**Comment by sacefe** [comment]
Changes completed

**Review by teraSurfer — APPROVED** [review]

**Pull request merged**